docs/manual/manual_redundant_throttle.rst

```diff
@@ -0,0 +1,113 @@
+
+The sequence diagram below illustrates how your module reacts to received throttle commands as well as how it deals with switching between sources.
+
+.. image:: ../_static/manual_images/redundant_throttle/iquart_prio_example.png
+    :width: 50%
+
+Note that the first DroneCAN throttle is applied to the motor since no IQUART messages had been received before. Then, when both an IQUART and DroneCAN message are received, 
+the IQUART throttle is applied. Last, DroneCAN throttles are only applied again once the throttle timeout is reached after IQUART disconnects. If for any reason, the module begins receiving 
+IQUART throttles again at this point, the module will immediately start accepting those as IQUART has a higher priority.
+
+Configuration Example 2
+##########################
+
+Suppose your module supports DroneCAN, IQUART, and hobby inputs. 
+
+You configure the following:
+
+- DroneCAN's priority to 3, IQUART to 0, hobby to 0
+- Flight controller outputting both :ref:`hobby_dshot` as well as DroneCAN commands
+- Throttle source timeout configured to 0.1s
+
+.. image:: ../_static/manual_images/redundant_throttle/dshot_ignored_ex.png
+    :width: 70%
+
+In this example, the module is configured to ignore all hobby and IQUART throttle messages. So, all received DSHOT throttles are dropped, and throttles are only applied once DroneCAN 
+throttles are received. An important note from this example is that the module will reach its :ref:`propeller motor control timeout <manual_timeout>` even if the 
+flight controller continues to send DSHOT throttles. As they are not processed by the throttle source manager, DSHOT throttles cannot be used to reset the timeout timer.
+
+********************************
+Redundant Throttle and Arming
+********************************
+
+All throttle commands received by the throttle source manager are sent to your module's :ref:`arming handler <manual_advanced_arming>`, and are subject to all constraints defined by the arming handler. 
+For example, suppose you have configured IQUART to be the highest priority source, and are sending IQUART throttle commands via the :ref:`Vertiq Testing Tool <vertiq_testing__guide>`. At the same 
+time, you are transmitting DroneCAN throttles with the :ref:`DroneCAN GUI tool <dronecan_gui_basics>`. In this example, your arming handler is set to arm on throttle in the range 0-7.5% on 10 
+consecutive throttles.
+
+VTT outputs IQUART throttles at 10%, and the DroneCAN GUI transmits 3% throttles. In this case, your module will not arm. Since IQUART has a higher priority, its throttles take 
+precedence over DroneCAN, so the arming handler will only receive the 10% throttles. If you pause the VTT output, however, your module will arm as the throttle source manager will 
+have transitioned to accepting the DroneCAN commands within the arming region. Now, if you unpause the VTT output, your module will start spinning at 10% throttle.

```

Comment:
I may add a bit more explanation at the end here to clarify a bit further for people who aren't as deep in the weeds as us. Something like:
"...within the arming region. Now that the module is armed, it will remain armed, even if the throttle source switches. If you unpause the VTT output your module will start spinning at 10% throttle because it is armed and receiving 10% throttle commands over IQUART, which have a higher priority than the DroneCAN throttle commands.